.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "loopback",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "rules": {
+    "max-len": ["error", 120, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
+      "ignorePattern": "^\\s*var\\s.=\\s*(require\\s*\\()|(/)"
+    }],
+    "camelcase": 0,
+    "one-var": "off",
+    "no-unused-expressions": "off",
+    "operator-linebreak": "off"
```

Comment:
Why do we have so many overrides? IMO, you should have NONE since you're startin ga new project -- use the defaults in eslint-config-loopback (we have overrides in other projects because its hard to refactor all at once, which doesn't apply in your case).